README.md

```diff
@@ -57,6 +57,18 @@ docker run -t -i -p 8080:8080 cockroachdb/cockroach shell
 # root@82cb657cdc42:/cockroach#
 ```
 
+If the docker command fails but Docker is installed, you probably need to initialize it.
+On OSX:
+```bash
+# Setup Boot2Docker. This should only need to be done once.
+boot2docker init
+# Start Boot2Docker. These commands will need to be run once per reboot.
+boot2docker start
+$(boot2docker shellinit)
```

Comment:
This needs to be run once per shell, which could be more than once per reboot depending on your workflow.